website/docs/components/card/card-api.md

```diff
@@ -5,7 +5,7 @@ tabs: Design('card'), A11y('card-a11y'), API('card-api'), Example('card-code'),
 
 ## Card
 
-It's just a `Box`.
+It's just a [Box](../../layout/box-system/box-api#box).
```

Comment:
Maybe we should add here something like: "It's just a [Box](../../layout/box-system/box-api#box) with predefined styles" — because it really has some styles already so that teams do not need to set all these shadows, borders, etc.

<img width="1470" alt="image" src="https://github.com/semrush/intergalactic/assets/77798437/613cfe2e-a17b-4244-8646-8288fbeb8606">

Reply:
Thanks, done! Also replaced "pre-installed" with "predefined", because I felt uneasy about that term, and the "pre-" prefix in particular (our and Google's guides don't mention that specific word).